### PR TITLE
specs: Update L2 Output RPC method

### PR DESCRIPTION
**Pull request by Inphi**
Fixes the outdated `optimism_outputAtBlock` specification.

### CODE REVIEW

File: specs/rollup-node.md [hunk 1]
```diff
@@ -61,6 +66,46 @@ byte hash corresponding to the [L2 output root](./proposals.md#l2-output-commitm
...
+- `timestamp`: `QUANTITY`, 64 Bits
+- `l1origin`: `BlockID`
+- `sequenceNumber`: `QUANTITY`, 64 Bits - distance to first block of epoch
+
+#### SyncStatus
+
+Represents a snapshot of the rollup driver.
+
+- `current_l1`: `Object` - instance of [`L1BlockRef`](#l1blockref).
+- `current_l1_finalized`: `Object` - instance of [`L1BlockRef`](#l1blockref).
```
Comment (ajsutton) [inline specs/rollup-node.md:100]: There's also `finalized_l1` which is different somehow and even the comment in the code doesn't really help me understand what's different about it.

File: specs/rollup-node.md [hunk 2]
```diff
@@ -72,7 +117,11 @@ The input and return types here are as defined by the [engine API specs][engine-
   1. `blockNumber`: `QUANTITY`, 64 bits - L2 integer block number.
 - returns:
   1. `version`: `DATA`, 32 Bytes - the output root version number, beginning with 0.
-  1. `l2OutputRoot`: `DATA`, 32 Bytes - the output root.
+  1. `outputRoot`: `DATA`, 32 Bytes - the output root.
+  1. `blockRef`: `Object` - instance of [`L1BlockRef`](#l1blockref).
```
Comment (ajsutton) [inline specs/rollup-node.md]: This appears to be a L2BlockRef - it includes a `l1origin` field and `sequencerNumber`

File: specs/rollup-node.md (hunk 1 above)
Comment (Inphi) [reply specs/rollup-node.md:100]: Somehow missed that. The `current_finalized_l1` is the derivation's own state of finalized L1. It's updated as derivation ingests blocks. Whereas, `finalized_l1` is the latest L1 state observed, but not processed, by derivation.

**Review by ajsutton — APPROVED** [review]
LGTM.

**Pull request merged**